src/herder/Herder.h

```diff
@@ -50,6 +50,9 @@ class Herder
     // state
     static std::chrono::seconds const SEND_LATEST_CHECKPOINT_DELAY;
 
+    // Timeout before publishing externalized values to archive
```

Comment:
I think it would be useful for nodes to opt out of this, so you could add a new config to the Config class instead of the const.